### PR TITLE
-Added a docker file that allows the building of go-mysqldump. Other …

### PR DESCRIPTION
**Pull request by michaelhutchful**
Inclusion of docker file to allow creation of docker image of go-mysqldump

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+ADD . /go/src/github.com/siddontang/go-mysql
+
+RUN apk add --no-cache mariadb-client
+
+RUN cd /go/src/github.com/siddontang/go-mysql/ && \
+    go build -o bin/go-mysqldump ./cmd/go-mysqldump && \
+    cp -f ./bin/go-mysqldump /go/bin/go-mysqldump
+
+ENTRYPOINT ["/sbin/tini","--","go-mysqldump"]
```
Comment (siddontang) [inline Dockerfile]: em, I think this may be a dockerfile for all the go-mysql binaries, not only for mysqldump.

**Comment by michaelhutchful** [comment]
Yes it is for all go-mysql binaries

Sent from my Windows 10 device

**Comment by siddontang** [comment]
@MichaelAtoHutchful 

but you only build go-mysqldump.

**Comment by GregoryIan** [comment]
@michaelhutchful  hi, you haven’t updated this pr for a long time, and I just want to know what is your plan now?

**Comment by michaelhutchful** [comment]
At the moment I don't have any plans to proceed with this.

**Pull request closed without merging**